RealtimeCSG/Assets/Plugins/RealtimeCSG/Editor/Scripts/View/GUI/EditModeGUI/EditModeToolWindow.Editor.cs

```diff
@@ -10,10 +10,7 @@ public class EditModeToolWindowEditor : Editor
 	public override void OnInspectorGUI()
 	{
 		if (EditorApplication.isPlayingOrWillChangePlaymode)
-		{
-			Selection.activeObject = null;
 			return;
-		}
```

Comment:
This shouldn't have been removed, this was a fix

Reply:
i thought i had changed that back. maybe i forgot to push the changes

Reply:
on checking this, i did indeed change it back, but git did not find the changes. should be fixed now